docs/multitenancy.md

```diff
@@ -0,0 +1,60 @@
+## Multitenancy
+
+### Overview
+
+Zenith supports multitenancy. One pagegserver can serve multiple tenants at once. Tenants can be managed via zenith CLI. During page server setup tenant can be created using ```zenith init --create-tenant``` Also tenants can be added into the system on the fly without pageserver restart. This can be done using the following cli command: ```zenith tenant create``` Tenants use random identifiers which can be represented as a 32 symbols hexadecimal string. So zenith tenant create accepts desired tenant id as an optional argument. The concept of timelines/branches is working independently per tenant.
+
+### Tenants in other commands
+
+Frequently tenant id apper in other commands, for example in branch create command: ```zenith branch ee6016ec31116c1b7c33dfdfca38891f new_one main``` creates branch `new_one` forked from `main` for tenant `ee6016ec31116c1b7c33dfdfca38891f`. This is also the case for internal page server interface used for communication with compute nodes. So here for example `pagestream` command will also require tenant id.
+
+Examples for cli:
+
+```sh
+zenith tenant list  
+
+zenith tenant create ee6016ec31116c1b7c33dfdfca38892f
+
+zenith pg create ee6016ec31116c1b7c33dfdfca38892f main
+
+zenith pg start ee6016ec31116c1b7c33dfdfca38892f main
+
+zenith pg stop ee6016ec31116c1b7c33dfdfca38892f main
+
+// lists branches for <tenantid>
+zenith branch ee6016ec31116c1b7c33dfdfca38892f
+```
```

Comment:
I think it is better to have default tenant and don't require tenant_id. If somebody wants to access non-default tenant he may specify --tenant